hasjob/views/helper.py

```diff
@@ -380,6 +380,27 @@ def get_post_viewcounts(jobpost_id):
     return values
 
 
+def get_max_counts(postids):
+    view_counts = [get_post_viewcounts(postid) for postid in postids]
```

Comment:
This is about 700 function calls in a loop (or how many ever jobs are currently listed). The slowdown will be noticeable. Please replace this with a loop through the source data structure.

Reply:
Can I cache this?